cluster/gce/gci/configure-helper-common.sh

```diff
@@ -597,7 +597,8 @@ function create-master-auth {
     append_or_replace_prefixed_line "${known_tokens_csv}" "${KUBE_BOOTSTRAP_TOKEN},"          "gcp:kube-bootstrap,uid:gcp:kube-bootstrap,system:masters"
   fi
   if [[ -n "${KUBE_CONTROLLER_MANAGER_TOKEN:-}" ]]; then
-    append_or_replace_prefixed_line "${known_tokens_csv}" "${KUBE_CONTROLLER_MANAGER_TOKEN}," "system:kube-controller-manager,uid:system:kube-controller-manager"
+    ## workaround for Arktos issue 1039, to treat the KCM token as admin
+    append_or_replace_prefixed_line "${known_tokens_csv}" "${KUBE_CONTROLLER_MANAGER_TOKEN}," "admin,admin,system:masters"
```

Comment:
Not sure how this would work. Also it does not look good for master

Reply:
this is not new in this PR. it is just a different way to workaround bug 1039 as it commented out. previously the workaround is to use the bootstrap kubeconfig.

Reply:
fixed